assemblers/org.osbuild.qemu

```diff
@@ -663,7 +663,8 @@ def main(tree, output_dir, options, loop_client):
             "qcow2": ["-c"],
             "vdi": [],
             "vmdk": ["-c"],
-            "vpc": ["-o", "subformat=fixed,force_size"]
+            "vpc": ["-o", "subformat=fixed,force_size"],
+            "vhdx": []
```

Comment:
`-c` (compress) is only documented to work with `qcow2` images. Are `vhdx` images always compressed or are the docs out of date and we need to pass it as well?

Reply:
`-c` doesn't work with `-O vhdx`, they are apparently always compressed.

Reply:
Thanks!